packages/mongo-nav/src/org-nav/OrgNav.spec.tsx

```diff
@@ -340,6 +340,78 @@ describe('packages/mongo-nav/src/org-nav', () => {
     });
   });
 
+  describe('the Access Manager dropdown displays correctly', () => {
+    test('when onPrem and currentProject exist, the Project Access Manager link is not disabled and project name is displayed', () => {
```

Comment:
Maybe helps to make a describe block with onPrem is true and false to bucket those two conditionals? These sentences are looong.

Reply:
👍